README.md

```diff
@@ -2,6 +2,10 @@
 
 A GTFS Schedule (static) [General Transit Feed Specification (GTFS)](https://gtfs.mobilitydata.org/spec/gtfs-schedule) feed validator
 
+This README contains information for master branch of this project, which is under active development. If you'd like to view documentation for past releases of the project, see:
+* [v1.4.0](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/README.md)
+* [v2.0.0](https://github.com/MobilityData/gtfs-validator/blob/v2.0.0/README.md)
+
 # Introduction
 This is a command-line tool written in Java that performs the following steps:
 1. Loads input GTFS zip file from a URL or disk
```

Comment:
In the "Run the app via command line" section of the README, the given command-line parameters won't work with the v2.0.0 release, right? Shouldn't we be referencing a snapshot JAR in the main README that's built from the master branch?

Reply:
Similar comment on the rest of the README that's still referencing v2.0.0